x/gov/init_test.go

```diff
@@ -66,7 +65,8 @@ func TestInitFromGenesis(t *testing.T) {
         "quorum": {
           "numerator": 2,
           "denominator": 3
-        }
+        },
+				"electorate_id": 2
```

Comment:
Whitespace in this file is off. How about pushing it through [`jq`](https://stedolan.github.io/jq/) to fix it?

Reply:
Guess it depends on tab/space ratio on your editor.
But I did fix as requested.